src/commons/services/utils.js

```diff
@@ -88,6 +90,25 @@
             });
         };
 
+        vm.getClusterDetails = function(clusterId) {
+            var cluster = {"name" : "Unassigned"};
+            if(vm.clusterData !== null && clusterId !== "") {
+                var i, key, clusterData, len;
```

Comment:
You can declare all the variables at the start of the function so that all the variable will be declared at single place and hence will be easy to manage.